SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -306,6 +306,23 @@ CEulerSolver::CEulerSolver(CGeometry *geometry, CConfig *config,
     DonorGlobalIndex[iMarker] = new unsigned long [nVertex[iMarker]]();
   }
 
+  /*--- Actuator Disk Radius allocation ---*/
+  ActDisk_R.resize(nMarker);
+
+  /*--- Actuator Disk Center allocation ---*/
+  ActDisk_C.resize(nMarker, MAXNDIM);
+
+  /*--- Actuator Disk Axis allocation ---*/
+  ActDisk_Axis.resize(nMarker, MAXNDIM);
+
+  /*--- Actuator Disk Fa, Fx, Fy and Fz allocations ---*/
+  for (iMarker = 0; iMarker < nMarker; iMarker++) {
+    ActDisk_Fa.resize(nMarker, nVertex[iMarker]);
+    ActDisk_Fx.resize(nMarker, nVertex[iMarker]);
+    ActDisk_Fy.resize(nMarker, nVertex[iMarker]);
+    ActDisk_Fz.resize(nMarker, nVertex[iMarker]);
+  }
```

Comment:
Sorry @EttoreSaetta, I think I gave you some unclear information, for the matrix type all rows need to have the same dimension, so for these 4 variables it would not work unfortunately.
For the other 3 variables it's fine, thanks for changing them.

Reply:
Yes, I am sorry, I have noticed it, and I am already fixing it.

Reply:
Did it, these 4 variables are not anymore `su2activematrix`, and I have restored the previous version for them.